docs/streams/upgrade-guide.html

```diff
@@ -108,6 +108,19 @@ <h3><a id="streams_api_changes_210" href="#streams_api_changes_210">Streams API
         they can override this config to a larger value.
     </p>
 
+    <p>
+        We've added an overloaded <code>StreamsBuilder#build</code> method that accepts an instance of <code>java.util.Properties</code> with the intent of using the
+        <code>StreamsConfig#TOPOLOGY_OPTIMIZATION</code> config added in Kafka Streams 2.0. Before 2.1, when building a topology with
+        the DSL, Kafka Streams writes the physical plan as the user makes calls on the DSL.  Now by passing in the <code>java.util.Properties</code> instance with a
+        <code>StreamsBuilder#build</code> call Kafka Streams can now optimize the physical plan of the topology, provided the <code>StreamsConfig#TOPOLOGY_OPTIMIZATION</code>
```

Comment:
`Now by passing in... can now optimize...` -- "now" is used repetitive.

Also: `passing in ... with a #build call` sounds weird?

`prided [that] the`